### PR TITLE
Adding libcereal-dev for non-boost serialization needs

### PR DESCRIPTION
**Pull request by pbeeson**
We use serialization for some storage of large data types, but are trying to move away from Boost depends.  Cereal is a substitute for boost serialization that is lightweight and beats boost in benchmarks.

https://packages.ubuntu.com/xenial/libcereal-dev
https://packages.debian.org/stretch/libcereal-dev
https://apps.fedoraproject.org/packages/cereal-devel

### CODE REVIEW

**Review by clalancette — CHANGES_REQUESTED** [review]
Can you please add in the links to the packages as requested in [CONTRIBUTING](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md)?  This just makes it easier for us to track the packages in the future.  Thanks.

**Comment by pbeeson** [comment]
CONTRIBUTING doesn't specific WHERE to add such links.  Presumably not in
the yaml.

**Comment by pbeeson** [comment]
https://packages.ubuntu.com/xenial/libcereal-dev
https://packages.debian.org/stretch/libcereal-dev
https://apps.fedoraproject.org/packages/cereal-devel

**Comment by clalancette** [comment]
> CONTRIBUTING doesn't specific WHERE to add such links. Presumably not in the yaml.

Yeah, just in the description of the PR is where we expect it.  An update to CONTRIBUTING to specify where to add the links would be appreciated.

**Review by clalancette — APPROVED** [review]

**Comment by pbeeson** [comment]
Just checking to see if there was anything else you were waiting on from me before this can get merged.  Thanks!

**Comment by clalancette** [comment]
> Just checking to see if there was anything else you were waiting on from me before this can get merged. Thanks!

Nope, just waiting for a second approval, then we can merge.

**Comment by pbeeson** [comment]
Not trying to be a jerk, but bumping this.  I have multiple entities waiting on a new feature, but I can't push to them until their install procedure automatically will install this system dep.

**Review by jacobperron — APPROVED** [review]

**Comment by pbeeson** [comment]
I appreciate the help.  Thanks

**Pull request merged**
